PixivImage.py

```diff
@@ -216,12 +216,12 @@ def ParseInfo(self, page):
 
         # datetime, in utc
         # "createDate" : "2018-06-08T15:00:04+00:00",
-        self.worksDateDateTime = datetime_z.parse_datetime(str(root["createDate"]))
+        self.worksDateDateTime = datetime_z.parse_datetime(root["createDate"])
         # Issue #420
         if self._tzInfo is not None:
             self.worksDateDateTime = self.worksDateDateTime.astimezone(self._tzInfo)
 
-        tempDateFormat = self.dateFormat or "%m/%d/%y %H:%M"  # 2/27/2018 12:31
+        tempDateFormat = self.dateFormat or "%Y%m%dT%H%M"     # 20180722T1231
```

Comment:
the date format is configurable in config.ini using `dateformat` and this supposed to be human readable

Reply:
The standard format is supposed to put in the date, not create random sub folders. Since we are fixing the separators we might as well fix the format, which is barely used anywhere in the world.
I agree with the human readability, but which format would be the best? ISO-like would be 2018-07-22 12:31, with the colons getting replaced in Windows. We could just go for the date without the time instead, which is what I think most people expect from a "work_date"-default anyway.

Reply:
`2018-07-22` if the use case to create folder based on worksdate is better.

Reply:
Ah, I'm sorry, I hadn't realized that this variable was used in other places (I had thought it was only for info/json info) - Sorry about that earlier suggestion.
The colon definitely wouldn't work then.